assets/locales/en/main.json

```diff
@@ -20,7 +20,8 @@
       "change": "Change width of a segment you’re pointing at<br />(hold <kbd class='key'>Shift</kbd> for more precision)"
     },
     "share": {
-      "sign-in": "<a href='/twitter-sign-in?redirectUri=/just-signed-in'>Sign in with Twitter</a> for nicer links to your streets and your personal street gallery",
+      "sign-in-twitter-link": "Sign In",
```

Comment:
This text should be "Sign in", with the lowercase `i`. [We have very strict guidelines about using sentence case for nearly all text](https://streetmix.readme.io/docs/content-style-guide#section-sentence-case).